snapper/Btrfs.h

```diff
@@ -59,9 +60,9 @@ namespace snapper
 	virtual SDir openInfosDir() const;
 	virtual SDir openSnapshotDir(unsigned int num) const;
 
-	virtual void createSnapshot(unsigned int num, unsigned int num_parent,
-				    bool read_only) const;
-	virtual void createSnapshotOfDefault(unsigned int num, bool read_only) const;
+	virtual void createSnapshot(unsigned int num, unsigned int num_parent, bool read_only,
```

Comment:
How about using C++11 'override' for those overwritten virtual methods? Even more so since the interface has now changed, so it would be desirable to make sure they are all overwritten correctly?